### ISSUE TITLE
Sacred Targe cubing problem

### ISSUE THREAD

**Issue by phenix1** [kolton/d2bot-with-kolbot#1423]
The script makes a hole in the shield paladin but it is probably the wrong ingredients into cubes
"Transmuting: Sacred Targe + Runa Ral + Runa Amn + Perfect Amethyst"
According to the formula should give Perfect Ruby
![error](https://user-images.githubusercontent.com/48789901/55637964-70da3300-57c6-11e9-9d6a-29d48eac2981.png)

**Comment by imbalanced** [comment]
Make sure you have correct recipe entry in your character config file.
`Config.Recipes.push([Recipe.Socket.Shield, "Sacred Targe", Roll.Eth]); // Socket ethereal Thresher`

Based on your log, my guess is that you have Recipe.Socket.Weapon configured instead of shield.

**Comment by phenix1** [comment]
Yes - my fault - thanks for your help and sorry for the confusion.
